app/src/main/res/layout/layout_todo_list_item.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/layout_todo_list_text"
+        android:layout_width="0dp"
+        android:layout_height="55dp"
+        android:layout_marginTop="7sp"
+        android:layout_marginEnd="7sp"
+        android:layout_weight="1"
+        android:fontFamily="sans-serif-black"
+        android:text="TextView"
```

Comment:
Not super sure about this, but I believe that if `"TextView"` is just a placeholder, we can just remove the whole `android:text` attribute. That way we avoid useless string (that are also considered hardcoded by the linter).